scripts/build_podspecs.sh

```diff
@@ -83,6 +83,11 @@ for target in "${targets[@]}"; do
     libraries="s.libraries = 'z'"
   fi
 
+  compiler_flags=""
+  if [[ "$target" == "CNIODarwin" ]]; then
+    compiler_flags="s.compiler_flags = '-D__APPLE_USE_RFC_3542=1'"
```

Comment:
@glbrntt if we set this unconditionally, why don't we just `#define __APPLE_USE_RFC_3542 1` before the `include` of the relevant header (that makes use of this)?

Reply:
Ah good point! I went for parity with the Package.swift but this is better, thanks for the suggestion.

Reply:
Then we can probably also just take it out of `Package.swift`, no?

Reply:
That's right, yes.

Unfortunately I can't get this to build this way -- perhaps there's a reason it was defined in Package.swift to begin with. Will have another look later.